src/values/expressions/TemplateLiteral.js

```diff
@@ -20,7 +20,7 @@ export default function extractValueFromTemplateLiteral(value) {
     if (type === 'TemplateElement') {
       return raw + part.value.raw;
     } else if (type === 'Identifier') {
-      return part.name === 'undefined' ? raw : `${raw}{${part.name}}`;
+      return part.name === 'undefined' ? `${raw}${part.name}` : `${raw}{${part.name}}`;
```

Comment:
I initially tried this myself but for `null` the `part.name` is actually `undefined` and the `part.value.raw` throws a TypeError so it seems that something is happening higher up already.